src/webportal/src/plugins/marketplace/template-list/template-list.item.component.ejs

```diff
@@ -1,15 +0,0 @@
-<% for(var i = 0, l = items.length; i < l; i += 1) { var item = items[i] %>
-<div class="content col-md-2 col-sm-4 col-xs-6" style="min-height: auto;">
-  <a href="?index=<%= this.pluginIndex %>&type=<%= this.parseType(item.type) %>&name=<%= encodeURIComponent(item.name) %>"
-    class="thumbnail" style="margin: 0">
-    <img src="/assets/img/marketplace/<%= this.parseType(item.type) %>.jpg" width="50%">
```

Comment:
These image paths are different from above ones, check whether we need to remove them